Index: festival/lib/languages.scm
===================================================================
--- festival.orig/lib/languages.scm	2009-02-20 08:48:45.000000000 -0600
+++ festival/lib/languages.scm	2009-02-20 08:51:36.000000000 -0600
@@ -101,6 +101,14 @@
    (Parameter.set 'Language 'czech)
 )
 
+(define (language_russian)
+"(language_russian)
+Set up language parameters for Russian."
+  (set! male1 voice_msu_ru_nsh_clunits)
+  (male1)
+  (Parameter.set 'Language 'russian)
+)
+
 (define (language_scots_gaelic)
 "(language_scots_gaelic)
 Set up language parameters for Scots Gaelic."
Index: festival/src/main/festival_main.cc
===================================================================
--- festival.orig/src/main/festival_main.cc	2009-02-20 08:53:40.000000000 -0600
+++ festival/src/main/festival_main.cc	2009-02-20 08:53:46.000000000 -0600
@@ -108,7 +108,7 @@
 	"              are printed (default if stdin not a tty)\n"+
 	"--language <string>\n"+
         "              Run in named language, default is\n"+
-	"              english, spanish and welsh are available\n"+
+	"              english, spanish, russian and welsh are available\n"+
 	"--server      Run in server mode waiting for clients\n"+
 	"              of server_port (1314)\n"+
 	"--script <ifile>\n"+
