Index: b/src/main/festival_main.cc
===================================================================
--- a/src/main/festival_main.cc
+++ b/src/main/festival_main.cc
@@ -44,6 +44,9 @@
 
 #include "festival.h"
 
+using std::cerr;
+using std::endl;
+
 static void festival_main(int argc, char **argv);
 static int festival_check_script_mode(int argc, char **argv);
 static void festival_script_mode(int argc, char **argv);
@@ -88,6 +91,8 @@
         "In evaluation mode \"filenames\" starting with ( are evaluated inline\n"+
 	"Festival Speech Synthesis System: "+ festival_version +"\n"+
 	"-q            Load no default setup files\n"+
+	"--datadir <string>\n"+
+	"              Set data directory pathname\n"+
 	"--libdir <string>\n"+
         "              Set library directory pathname\n"+
         "-b            Run in batch mode (no interaction)\n"+
@@ -127,6 +132,12 @@
 	festival_libdir = wstrdup(al.val("--libdir"));
     else if (getenv("FESTLIBDIR") != 0)
 	festival_libdir = getenv("FESTLIBDIR");
+
+    if (al.present("--datadir"))
+	festival_datadir = wstrdup(al.val("--datadir"));
+    else if (getenv("FESTDATADIR") != 0)
+	festival_datadir = getenv("FESTDATADIR");
+
     if (al.present("--heap"))
 	heap_size = al.ival("--heap");
 
@@ -233,7 +244,7 @@
 
     if (argc < 2)
     {   
-	cerr << "festival: script_mode has no file to interpret" << endl;
+	std::cerr << "festival: script_mode has no file to interpret" << std::endl;
 	return;
     }
 
