Index: b/src/arch/festival/ngram.cc
===================================================================
--- a/src/arch/festival/ngram.cc
+++ b/src/arch/festival/ngram.cc
@@ -40,6 +40,9 @@
 #include "festival.h"
 #include "festivalP.h"
 
+using std::cout;
+using std::endl;
+
 static LISP ngram_loaded_list = NIL;
 static EST_Ngrammar *load_ngram(const EST_String &filename);
 static LISP add_ngram(const EST_String &name,EST_Ngrammar *n);
