Index: b/src/modules/base/pos.cc
===================================================================
--- a/src/modules/base/pos.cc
+++ b/src/modules/base/pos.cc
@@ -41,6 +41,9 @@
 #include "festival.h"
 #include "lexicon.h"
 
+using std::cerr;
+using std::endl;
+
 static EST_VTCandidate *pos_candlist(EST_Item *s,EST_Features &f);
 static EST_VTPath *pos_npath(EST_VTPath *p,EST_VTCandidate *c,EST_Features &f);
 static double find_np_prob(EST_VTPath *p,int n,int *state);
