Index: b/src/modules/UniSyn_phonology/UniSyn_build.cc
===================================================================
--- a/src/modules/UniSyn_phonology/UniSyn_build.cc
+++ b/src/modules/UniSyn_phonology/UniSyn_build.cc
@@ -44,6 +44,8 @@
 #include <fstream>
 #include "festival.h"
 
+using namespace std;
+
 #include "../UniSyn/us_features.h"
 
 void merge_features(EST_Item *from, EST_Item *to, int keep_id);
